egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -234,6 +242,9 @@ def __init__(
     ):
         super(ScaledConv1d, self).__init__(*args, **kwargs)
         initial_scale = torch.tensor(initial_scale).log()
+
+        self.bias_scale: Optional[nn.Parameter]  # for torchscript
```

Comment:
I did not include this line, it did not raise any error.

Reply:
It raised an error last night. I just tried again by removing it and it's strange there are no errors. You can remove it.